### ISSUE TITLE
TypeError: this.$slots.default is not a function

### ISSUE THREAD

**Issue by Sky-Cat** [mercs600/vue3-perfect-scrollbar#20]
Vue: 3.2.47
vue3-perfect-scrollbar: 1.6.1

Error:
![изображение](https://user-images.githubusercontent.com/5306467/228636944-71e09760-b512-4fda-86d5-d6f6e085b3da.png)

in this function:

```js
render () {
    return h(this.tag,
      {
        class: 'ps'
      },
      this.$slots.default && this.$slots.default())
  }
```

**Comment by kindbuds** [comment]
same:

**Comment by saug19** [comment]
Same

**Comment by LevDomasnih** [comment]
same

**Comment by medzhidov** [comment]
Who has the same problem, just use my snippet as a scrollbar component:

https://gist.github.com/medzhidov/3d66e1db7b9b4d61dcb941ca58668e3c

**Comment by mercs600** [comment]
it should be resolved with version v2.0.0
